jscomp/others/js_json.ml

```diff
@@ -60,7 +61,28 @@ let test (type a) (x : 'a) (v : a kind) : bool =
   | Array -> Js_array.isArray x 
   | Object -> (Obj.magic x) != Js.null && Js.typeof x = "object" && not (Js_array.isArray x )
 
-
 external parse : string -> t = "JSON.parse" [@@bs.val]
 external stringifyAny : 'a -> string option = "JSON.stringify" [@@bs.val] [@@bs.return undefined_to_opt]
 (* TODO: more docs when parse error happens or stringify non-stringfy value *)
+
+let null : t = (Obj.magic (Js.null: 'a Js.null) : t)
```

Comment:
@bobzhang this does not work anymore, it's now an empty object not a `null` value.